src/components/application_manager/src/commands/mobile/register_app_interface_request.cc

```diff
@@ -590,9 +590,9 @@ void RegisterAppInterfaceRequest::SendRegisterAppInterfaceResponseToMobile() {
                                             application->mac_address());
   }
 
+  SendResponse(true, result_code, add_info.c_str(), &response_params);
   SendOnAppRegisteredNotificationToHMI(
       *(application.get()), resumption, need_restore_vr);
-  SendResponse(true, result_code, add_info.c_str(), &response_params);
```

Comment:
@jacobkeeler I don't know any reason to notify HMI first, so if you did not find strict requirement for that it could be changed